noxfile.py

```diff
@@ -98,7 +98,10 @@ def tests(session: PowerSession, coverage, pkg_specs):
     conda_prefix = Path(session.bin)
     if conda_prefix.name == "bin":
         conda_prefix = conda_prefix.parent
-    session.run2("conda list", env={"CONDA_PREFIX": str(conda_prefix), "CONDA_DEFAULT_ENV": session.get_session_id()})
+    try:
+        session.run2("conda list", env={"CONDA_PREFIX": str(conda_prefix), "CONDA_DEFAULT_ENV": session.get_session_id()})
+    except CommandFailed:
```

Comment:
This i've added mostly to myself, becouse i use nox without conda like this:

```
nox -s tests-3.9 --default-venv-backend venv
```

Reply:
I see your point (and for some other projects I do not use conda), however for consistency with this repo's state I'll prefer that we stick to the current usage of conda. Therefore I'll suggest to remove this mod from this PR